RecoHI/HiMuonAlgos/python/HiRegitMuonPixelPairStep_cff.py

```diff
@@ -52,7 +52,7 @@
     clusterCheck = "hiRegitMuClusterCheck",
 )
 hiRegitMuPixelPairStepSeeds     = RecoTracker.IterativeTracking.PixelPairStep_cff.pixelPairStepSeeds.clone(
-    seedingHitSets = "hiRegitMuPixelPairStepHitDoublets"
+    seedingHitSets = cms.InputTag("hiRegitMuPixelPairStepHitDoublets")
```

Comment:
was this type specialization required?
It's better not to specify the type if the parameter is meant to be defined already